### PR TITLE
fixing issue #5891 of Itseez/opencv.

### LINKED ISSUE #5891: loading SVM model not supported in python
I'm using OpenCV 3.0-beta and it seems that, according to my experience with the ml python module and various sites (e.g. http://answers.opencv.org/question/75077/opencv-300-python-svmload-is-missing/), the possiblity to load a serialized and stored svm from disk is not properly wrapped in the python bindings.

The issue could be related to #4969 or even be the same.

Is there someone currently working on this issue?

### PR DESCRIPTION
**Pull request by DarwinsBuddy**
Someone forgot to wrap the load function for SVMs in the
corresponding ml python module. Hopefully fixed that.
fixes #5891

### CODE REVIEW

File: modules/ml/src/svm.cpp [hunk 1]
```diff
@@ -2261,6 +2261,17 @@ Ptr<SVM> SVM::create()
...
 }
 
+Ptr<SVM> SVM::load(const String& filename)
+{
+    FileStorage fs;
+    fs.open(filename, FileStorage::READ);
+
+    Ptr<SVM> svm = makePtr<SVMImpl>();
+
+    svm->read(fs.getFirstTopLevelNode());
```
Comment (alalek) [inline modules/ml/src/svm.cpp]: Try to use here something like this `((SVMImpl*)svm.get())->read(...)` and remove virtual `read` method (ABI reports about vtable modification)

Comment (DarwinsBuddy) [inline modules/ml/src/svm.cpp]: Thank you very much for potining out this issue.
I'm going to fix this issue asap. :+1:

**Comment by DarwinsBuddy** [comment]
The build error related to my last commit seems to have no relation to my changes.
I would appreciate if someone could explain me what I've done wrong. :confused:

**Comment by alalek** [comment]
:+1:

**Comment by lonewolf9277** [comment]
Is this issue fixed now?

**Comment by DarwinsBuddy** [comment]
i think so.
@alalek would you confirm?

**Pull request merged**